### PR TITLE
A bunch of editorial changes to the security considerations suggested

### PR DESCRIPTION
**Pull request by ekr**
by Hugo Krawczyk.

### CODE REVIEW

File: draft-ietf-tls-tls13.md [hunk 1]
```diff
@@ -4535,6 +4537,9 @@ which PSK the server ultimately selects. The client will need to compute
 one for each potential PSK; if no PSK is selected, it will then need to
 compute the early secret corresponding to the zero PSK.
 
+Once all the values which are to be derived from a given secre<t have
```
Comment (davegarrett) [inline draft-ietf-tls-tls13.md]: typo: a wild angle bracket appeared

File: draft-ietf-tls-tls13.md [hunk 2]
```diff
@@ -5372,14 +5378,18 @@ to define a new exporter that includes the full handshake
 transcript.
 
 For all handshake modes, the Finished MAC (and where present, the
-signature), prevents downgrade attacks. In addition, the use of
+signature), prevents downgrade attacks
+In addition, the use of
```
Comment (davegarrett) [inline draft-ietf-tls-tls13.md]: accidental change? missing period followed by capitalization

File: draft-ietf-tls-tls13.md [hunk 3]
```diff
@@ -5489,6 +5515,12 @@ a separate nonce for each record, with the nonce being derived from
 the record sequence number ({{nonce}}), with the sequence
 number being maintained independently at both sides thus records which
 are delivered out of order result in AEAD deprotection failures.
+In order to prevent mass cryptanalysis when the same plaintext is
+repeatedly encrypted by different users under the same key
+(as is commonly the case for HTTP), the nonce is formed by mixing
+the sequence number with a secret per-connection initialization
+vector derived along with the traffic keys.
+See {{BT16}} for analysis of this construction,
```
Comment (davegarrett) [inline draft-ietf-tls-tls13.md]: the ending punctuation here should be a period, not a comma

**Comment by kaduk** [comment]
Other than davegarrett's editorial notes, this looks good.

File: draft-ietf-tls-tls13.md (hunk 2 above)
Comment (ekr) [inline draft-ietf-tls-tls13.md]: yeah.  I meant to add a cite and then ended up putting it later.

**Pull request merged**